src/solvers/smt/smt_conv.cpp

```diff
@@ -1181,6 +1181,25 @@ smt_astt smt_convt::convert_ast(const expr2tc &expr)
     a = mk_extract(a, ex.upper, ex.lower);
     break;
   }
+  case expr2t::code_comma_id:
+  {
+    // the expression on the right side will become
+    // the value of the entire comma-separated expression.
+    // e.g.
+    //    res = side_1, side_2;
+    // equals to
+    //    side1;
+    //    res = side_2;
+    // thus we convert the two operands(side1, side2) individually and sequentially
+    const code_comma2t &cm = to_code_comma2t(expr);
+    a = convert_ast(cm.side_1);
```

Comment:
What's the purpose of calling `convert_ast` on `cm.side_1` and discarding the result?

Reply:
Deleted.